WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostListFragment.java

```diff
@@ -84,6 +84,10 @@
         implements ReaderInterfaces.OnPostSelectedListener,
                    ReaderInterfaces.OnTagSelectedListener,
                    ReaderInterfaces.OnPostPopupListener {
+    private static final String READER_DETAIL_TYPE_KEY = "post-detail-type";
+    private static final String READER_DETAIL_TYPE_NORMAL = "normal";
+    private static final String READER_DETAIL_TYPE_BLOG_PREVIEW = "preview-blog";
+    private static final String READER_DETAIL_TYPE_TAG_PREVIEW = "preview-tag";
```

Comment:
I'm not sure how we're handling similar constants, but my vote would be to move these to the `Analytics` package so we don't have analytics-related strings strewn about the code.